packages/@aws-cdk/aws-apigateway/lib/deployment.ts

```diff
@@ -1,4 +1,4 @@
-import cdk = require('@aws-cdk/cdk');
+import { Construct, DeletionPolicy, Resource, Token } from '@aws-cdk/cdk';
```

Comment:
I personally don't like doing that, and it introduces a large amount of needless touches in those files...

Reply:
OK.